src/rdf4cpp/rdf/datatypes/registry/DatatypeConversion.hpp

```diff
@@ -8,13 +8,12 @@
 #include <type_traits>
 #include <utility>
 
+#include <rdf4cpp/rdf/datatypes/registry/util/Tuple.hpp>
```

Comment:
Tuple doesn't seem to be required here

Reply:
The file is using util::tuple_map, so that is required

Reply:
I must have overlooked tuple_map. Never mind and keep it.